api/src/migrations/extract-transition-seq.js

```diff
@@ -0,0 +1,85 @@
+//
+// Uses the same logic we used to use at runtime to find the existing metadoc, and then convert into
+// it into the new singular value style.
+//
+const db = require('../db');
+const logger = require('../logger');
+
+const TRANSITION_SEQ_DOCUMENT = '_local/transitions-seq';
+const METADATA_DOCUMENT = '_local/sentinel-meta-data';
+const OLD_METADATA_DOCUMENT = 'sentinel-meta-data';
+
+const migrateOldMetaDoc = doc => {
```

Comment:
this code is written weirdly, because it's an intentional copy of the live code, as opposed to something smarter or cleaner but with different logic. We can clean it up if we feel like that is important.